docs/developing-contracts/examples/Dutch Auction.md

```diff
@@ -0,0 +1,119 @@
+---
+sidebar_label: 'Dutch Auction'
+sidebar_position: 10
+---
+
+# Dutch auction
+
+## Introduction
+A Dutch auction is one of several types of auctions for buying or selling goods. Most commonly, it means an auction in which the auctioneer begins with a high asking price in the case of selling, and lowers it until some participant accepts the price, or it reaches a predetermined reserve price. A Dutch auction has also been called a clock auction or open-outcry descending-price auction. This type of auction shows the advantage of speed since a sale never requires more than one bid.
+
+## Prerequisites
+[Gear ERC-721](https://wiki.gear-tech.io/developing-contracts/examples/erc-721#erc-721-interface)
```

Comment:
Pls clarify - Prerequisites Gear ERC-721 - looks confusing.
Btw, we now write fungible/non-fungible tokens, ERC - is the Ethereum's standard (we can also consider calling it - GRC-721).